.gitignore

```diff
@@ -23,7 +23,8 @@ yarn-debug.log*
 yarn-error.log
 # Local config
 /config/local.js
-/src/config/local.js
+src/config/local.js
 
 #Other
+yarn.lock
```

Comment:
We should be committing this. The lockfile ensures everyone is using the same version of dependencies. I had thought when package-lock.json was added here it was because we were using yarn. We definitely need to have the lockfile for whichever of these we're using.

Just a guess, but if you've added these because they get annoying merge conflicts sometimes, a quick way to deal with them is just to reset the lockfile to the last known good state and then run yarn install again. So you'd do:

`git checkout -- yarn.lock`
then
`yarn install`

and any merge conflicts should be fixed